post-processor/shell-local/post-processor.go

```diff
@@ -176,9 +180,25 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 			return nil, false, fmt.Errorf("Error processing command: %s", err)
 		}
 
+		ui.Say(fmt.Sprintf("Making script executable"))
+		if runtime.GOOS == "windows" {
+			err := os.Chmod(script, 0600) // read and write perms. Should be enough.
+			if err != nil {
+				fmt.Println(err)
+			}
+		} else {
+			err := os.Chmod(script, 0777)
```

Comment:
0744 because we don't want these to be group/world executable

Reply:
Cool.

Reply:
Yeah, meant to fix this post-debugging-frenzy, and, of course, forgot.